src/modules/dataFetching/components/GenericTableWithData.tsx

```diff
@@ -152,6 +152,11 @@ const GenericTableWithData = <
     )
   );
 
+  // if the filters change, return to the first page
+  useEffect(() => {
+    setPage(0);
+  }, [filterValues]);
```

Comment:
global behavior change, does this seem okay?

Reply:
yep, thanks